### ISSUE TITLE
Godot crash on start in Mac OS X 10.10.3

### ISSUE THREAD

**Issue by almindor** [godotengine/godot#2301]
SIGSEGV right on start with official 1.1 app.

ERROR: create: Condition ' !create_func ' is true. returned: 0
   At: core/os/mutex.cpp:39.
Segmentation fault: 11

**Comment by tyoc213** [comment]
Same here

**Comment by d3v0x** [comment]
If you're using multiple monitors, drag the application (focus) to the main monitor before starting. This works for me (starting from command line but it should work with the launcher as well)

**Comment by tyoc213** [comment]
Yes, at less I'm with 2 monitors I was launching it on the second one.

**Comment by d3v0x** [comment]
So I think it's the same like https://github.com/okamstudio/godot/issues/2216

**Comment by tyoc213** [comment]
Yes, it is, after do that I get

```
$ bin/godot.osx.tools.32 
ERROR: create: Condition ' !create_func ' is true. returned: 0
   At: core/os/mutex.cpp:39.
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
nscreen count 2
added screen 1
added screen 2
```

**Comment by vnen** [comment]
Closing this as duplicate of #2216.
